### DISCUSSION TITLE
Confused about how to use getInputProps

### DISCUSSION THREAD

**Discussion by OhYello** [react-dropzone/react-dropzone#1281] [General]
This is what my current <input> looks like and I'm confused on how to translate this to getInputProps. Any help would be appreciated, thanks for reading!

```
<input
  id="file-upload"
  type="file"
  accept="image/x-png,image/gif,image/jpeg"
  onChange={onSelectFile}
  ref={selectFileRef}
  onClick={() => selectFileRef.current?.click()}
/>
```

**Comment by rolandjitsu** [comment]
@OhYello please checkout https://react-dropzone.js.org/#usage.
